README.md

```diff
@@ -2,8 +2,6 @@
 
 <a name="readme-top">Hello Microverse</a>
 
-
-
 <!-- TABLE OF CONTENTS -->
 
```

Comment:
- [ ] Just as the previous reviewer mentioned, kindly add all the missing sections as pointed out. I've attached a screenshot of how your header should look like. do make the changes, and make the README Lok for professional. You've don a good job so far 👏 

## Expected content 👍
![image](https://user-images.githubusercontent.com/50990925/212281787-80c809c6-02b9-421c-ac05-9f823a293e75.png)